packages/flutter/test/material/text_field_test.dart

```diff
@@ -250,6 +250,75 @@ void main() {
     await checkCursorToggle();
   });
 
+  testWidgets('cursor has expected defaults', (WidgetTester tester) async {
+    await tester.pumpWidget(
+        overlay(
+          child: const TextField(
+            cursorRadius: Radius.circular(3.0),
+          ),
+        )
+    );
+
+    final TextField textField = tester.firstWidget(find.byType(TextField));
+    expect(textField.cursorWidth, 2.0);
+    expect(textField.cursorRadius, const Radius.circular(3.0));
+  });
+
+  testWidgets('cursor layout has correct width', (WidgetTester tester) async {
+    final TextEditingController controller = new TextEditingController();
+
+    await tester.pumpWidget(
+        overlay(
+          child: new RepaintBoundary(
+            key: const ValueKey<int>(1),
+            child: new TextField(
+              controller: controller,
+              cursorWidth: 15.0,
+            ),
+          ),
+        )
+    );
+    expect(controller.selection.baseOffset, -1);
+    expect(controller.selection.extentOffset, -1);
+
+    const String testValue = ' ';
+    await tester.enterText(find.byType(TextField), testValue);
+    await skipPastScrollingAnimation(tester);
+
+    await expectLater(
+      find.byKey(const ValueKey<int>(1)),
```

Comment:
Wouldn't find.byType(TextField) be adequate?

Reply:
Yes, changed.